src/tests/EdFi.DataManagementService.Tests.E2E/Features/UpdateInvalidReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,203 @@
+                  }
+                  """
+
+        @ignore
+        Scenario: Ensure clients cannot update a resource using a correct reference but missing the other one
+             When a PUT request is made to "ed-fi/studentEducationOrganizationAssociations/1b57dcf9955f4bf987cebea2075c0ed6" with
+                  """
+                  {
+                      "studentReference": {
+                        "studentUniqueId":"604834"
+                      }
+                  }
+                  """
+             Then it should respond with 409
+              And the response body is
+                  """
+                  {
+                      "detail": "The referenced 'EducationOrganizationReference' item does not exist.",
+                      "type": "urn:ed-fi:api:data-conflict:unresolved-reference",
+                      "title": "Unresolved Reference",
+                      "status": 409,
+                      "correlationId": null
+                  }
+                  """
+
+        @ignore
+        Scenario: Ensure clients cannot update a resource that uses a reference more than once
+             When a PUT request is made to "ed-fi/studentCTEProgramAssociations/9aa43c870d244c939d366fbfd5ecab10" with
+                  """
+                  {
+                      "beginDate": "2020-06-05",
+                      "programReference": {
+                        //"educationOrganizationId": 255901,
+                        "programName": "Career and Technical Education",
+                        "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Career and Technical Education"
+                      },
+                      "studentReference": {
+                          "studentUniqueId": "604825"
+                      }
+                  }
+                  """
+             Then it should respond with 409
+              And the response body is
+                  """
+                  {
+                      "detail": "The referenced 'EducationOrganizationReference' item does not exist.",
+                      "type": "urn:ed-fi:api:data-conflict:unresolved-reference",
+                      "title": "Unresolved Reference",
+                      "status": 409,
+                      "correlationId": null
+                  }
+                  """
+
+        @ignore
+        Scenario: Ensure clients cannot update a resource that uses a reference more than once and misses another required reference
+             When a PUT request is made to "ed-fi/studentCTEProgramAssociations/9aa43c870d244c939d366fbfd5ecab10" with
+                  """
+                  {
+                      "beginDate": "2020-06-05",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 255901
+                      },
+                      "programReference": {
+                        //"educationOrganizationId": 255901,
```

Comment:
Same as previous comment, not sure if this commented line is required.

Reply:
Removed